### ISSUE TITLE
Probably false-positive for E999 in Python 3.11.1

### ISSUE THREAD

**Issue by Divaltor** [charliermarsh/ruff#2571]
Hello, in Python 3.11.1 with enabled E999 rule ruff throw error on code with catching new exception groups

```python
try:
   ...
except* ValueError as ex:
   ...
```

```bash
$ ruff .
> E999 SyntaxError: invalid syntax. Got unexpected token '*'
```

In settings is set `target-version = "py311"`

**Comment by charliermarsh** [comment]
Yeah good catch. We'll need to fix this upstream in RustPython.

**Comment by youknowone** [comment]
https://github.com/RustPython/RustPython/issues/4479
